librz/core/cmd/cmd_debug.c

```diff
@@ -3904,6 +3820,50 @@ RZ_IPI int rz_cmd_debug_continue_until(void *data, const char *input) {
 	return RZ_CMD_STATUS_OK;
 }
 
+// dor
+RZ_IPI int rz_cmd_debug_process_dor(void *data, const char *input) {
```

Comment:
Ideally, you could try to use the same approach for `k=v` as in `e` command, for example, see the corresponding code. This way we could remove the need for OLDINPUT here as well.